=== modified file 'CMakeLists.txt'
--- CMakeLists.txt	2010-04-20 23:20:46 +0000
+++ CMakeLists.txt	2010-04-22 11:30:21 +0000
@@ -276,17 +276,17 @@
 execute_process(
 	COMMAND libetpan-config --cflags
 	OUTPUT_VARIABLE MAIL_PACKAGE_CFLAGS)
-STRING (REGEX REPLACE "-I" "" MAIL_PACKAGE_INCLUDE_DIRS "${MAIL_PACKAGE_CFLAGS}")  # salete de libetpan ...
 execute_process(
 	COMMAND libetpan-config --libs
 	OUTPUT_VARIABLE MAIL_PACKAGE_LIBS)
-STRING (REGEX REPLACE "\n" "" MAIL_PACKAGE_LIBS "${MAIL_PACKAGE_LIBS}")
 #STRING (REGEX REPLACE "-L([^ ]+).*" "-L\\1" MAIL_PACKAGE_LIBRARY_DIRS ${MAIL_PACKAGE_LIBS})
 #STRING (REGEX REPLACE "${MAIL_PACKAGE_LIBRARY_DIRS} " "" MAIL_PACKAGE_LIBRAIRIES ${MAIL_PACKAGE_LIBS})
 
 if ("${MAIL_PACKAGE_LIBS}" STREQUAL "")
 	message (STATUS "warning : Could not find libetpan; Cairo-Dock won't be built with Mail applet.")
 else()
+	STRING (REGEX REPLACE "-I" "" MAIL_PACKAGE_INCLUDE_DIRS "${MAIL_PACKAGE_CFLAGS}")  # salete de libetpan ...
+	STRING (REGEX REPLACE "\n" "" MAIL_PACKAGE_LIBS "${MAIL_PACKAGE_LIBS}")
 	set (GETTEXT_MAIL ${GETTEXT_PLUGINS})
 	set (VERSION_MAIL "1.0.3")
 	set (PACKAGE_MAIL "cd-mail")

